index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>themeWagon</title>
+		<!-- <link rel="stylesheet" href="../task-html-and-css-basics/reset.css" /> -->
+		<script
+			src="https://kit.fontawesome.com/02b621b881.js"
+			crossorigin="anonymous"
+		></script>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="../task-html-and-css-basics/style.css" />
```

Comment:
Raczej poprawny adres to `./style.css` - ścieżki piszmy względem głównego katalogu, a "ponad".

Ta zmiana spowoduje, żę inne obrazki nie będą działać - trzeba by było je poprawić.
Raczej standardowo, pobierając repo to uruchamiamy jego zawartość względem tego katalogu z repo tj. `task-html-and-css-basics` - niestety w obecnej wersji to nie zadziała.